### DISCUSSION TITLE
How to trigger a new release for symfony/http-client?

### DISCUSSION THREAD

**Discussion by ersatzhero** [symfony/symfony#49872] [Q&A] [answered]
Hello,

there are several bugfixes in symfony/http-client, which I need in my project.
How can I trigger a new build, so there is a new tag with all bugfixes in it?

Kind Regards

**Comment by ker0x** [comment, marked as answer]
You can't, you will have to wait until a new patch version is release, which should happened tomorrow or the next week!

[Symfony release process](https://symfony.com/doc/current/contributing/community/releases.html)
